android/capacitor/src/main/java/com/getcapacitor/JSInjector.java

```diff
@@ -71,10 +71,15 @@ public String getScriptString() {
     public InputStream getInjectedStream(InputStream responseStream) {
         String js = "<script type=\"text/javascript\">" + getScriptString().replace("${", "\\${") + "</script>";
```

Comment:
```suggestion
        String js = "<script type=\"text/javascript\">" + getScriptString() + "</script>";
```
This can be removed since it was added for replaceFirst to work, since the PR removes the replaceFirst it's not needed